edx_solutions_api_integration/permissions.py

```diff
@@ -86,6 +87,23 @@ def has_permission(self, request, view):
             return True
 
 
+class IsStaffOrEnrolled(permissions.BasePermission):
+    """
+    Permission that allows access to admin users or the enrolled users of a course.
+    """
+    def has_permission(self, request, view):
+        user = request.user
+        course_id = request.parser_context.get('kwargs', {}).get('course_id', None)
+
+        if course_id:
+            course_key = get_course_key(course_id)
```

Comment:
Can we move this statement `course_key = get_course_key(course_id)`
above `If` statement and then only check `if course_key`